wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -545,6 +545,14 @@ abstract class Wallet
     } yield tx
   }
 
+  override def sweepWallet(address: BitcoinAddress, feeRate: FeeUnit)(implicit
+      ec: ExecutionContext): Future[Transaction] = {
+    for {
+      outpoints <- listUtxos().map(_.map(_.outPoint))
+      tx <- sendFromOutPoints(outpoints, address, feeRate)
+    } yield tx
```

Comment:
nit: Maybe worth adding a `logger.info(s"Sweeping wallet balance=${balance} to address=${address}")`